### PR TITLE
[FRAF-772] Add message prop in Select for replacing default "No options" label

### PR DESCRIPTION
**Pull request by eniskraasniqi1**
### Description

The default "No options" Label is not translatable, so I added a prop which accepts a string and replaces the default one.

### CODE REVIEW

**Review by lorgan3 — CHANGES_REQUESTED** [review]
B: All rest props are already passed to the select so this was already possible.
It's fine to explicitly document this but you shouldn't change the type (from function to string)

You can find some examples here https://github.com/search?q=org%3Ateamleadercrm+noOptionsMessage&type=code

**Comment by eniskraasniqi1** [comment]
> B: All rest props are already passed to the select so this was already possible. It's fine to explicitly document this but you shouldn't change the type (from function to string)
> 
> You can find some examples here https://github.com/search?q=org%3Ateamleadercrm+noOptionsMessage&type=code

Oh shit, you're absolutely right... Thank you, will close this one 🙏

File: src/components/select/Select.js [hunk 1]
```diff
@@ -363,8 +363,19 @@ class Select extends PureComponent {
   });
 
   render() {
-    const { components, creatable, error, inverse, helpText, size, success, warning, forwardedRef, ...otherProps } =
-      this.props;
+    const {
+      components,
+      noOptionsMessage = 'No options',
```
Comment (jelledc) [inline src/components/select/Select.js:368]: PP: I would drop the default value as it is already the default in `react-select`.

```suggestion
      noOptionsMessage,
```

**Review by jelledc — APPROVED** [review]
~Looks good, did a manual check and it works as intended.~ Ticket was closed in the mean time :see_no_evil: .

**Pull request closed without merging**